src/cli_config.rs

```diff
@@ -32,6 +32,8 @@ impl FromStr for ThreadsConfig {
 /// Config parsed from command line arguments.
 #[derive(Clone, PartialEq, Eq, Debug)]
 pub struct CliConfig {
+    /// Path to the file where DHT keys are stored.
+    pub keys_file: String,
```

Comment:
Maybe better use Path?

Reply:
And, is there no Operating system or file system dependency?

Reply:
No

Reply:
(Path)[https://doc.rust-lang.org/nightly/std/path/struct.Path.html]  can't be stored in a struct since it's unsized.

Reply:
OK. Let it be String